types/canvasjs/canvasjs-tests.ts

```diff
@@ -1,248 +1,246 @@
+import * as CanvasJS from "canvasjs";
```

Comment:
I dont see this as the import style shown. https://www.npmjs.com/package/canvasjs#getting-started shows it to be:
```TypeScript
imported as import CanvasJs from 'canvasjs'
```